disspy_v2/app.py

```diff
@@ -57,19 +59,16 @@ def __init__(self, data: dict, token: str, session, hook) -> None:
 
     async def _respond(self, payload: dict):
         _token, _id = self._interaction.token, self._interaction.id
-        _url = f'https://discord.com/api/v10/interactions/{_id}/{_token}/callback'
-        async with self._session.post(_url, data=dumps(payload)) as r:
-            try:
-                return await r.json()
-            except ContentTypeError:
-                return await r.text()
+        route = Route('/interactions/%s/%s/callback', _id, _token, method='POST', token=self._token, payload=payload)
+        j = await route.async_request(self._session, get_event_loop())
+        return j
 
     async def send_message(self, *strings: list[str], sep: str = ' ', ephemeral: bool = False):
         await self._respond({
-            'type': 4,
+            'type': InteractionCallbackType.channel_message_with_source,
             'data': {
                 'content': str(utils.get_content(*strings, sep=sep)),
-                'flags': 1 << 6 if ephemeral else 0
+                'flags': MessageFlags.ephemeral if ephemeral else 0
```

Comment:
```suggestion
                'flags': MessageFlags.ephemeral if ephemeral else _typing.empty
```
And create new module _typing.py